internal/commands/app/diff.go

```diff
@@ -97,10 +97,12 @@ func (cmd *CommandDiff) Handler(profile *user.Profile, ui terminal.UI, clients c
 	}
 
 	if cmd.inputs.IncludeDependencies {
-		uploadPath, err := local.PrepareDependencies(app, ui)
+		uploadDependencies, err := local.FindAppDependencies(app.RootDir)
 		if err != nil {
 			return err
 		}
+		uploadPath := uploadDependencies.ArchivePath
```

Comment:
I don't think we need to declare this as its own `var` ... also I don't think we want to be calling `defer os.Remove(uploadPath)` down below anymore ... that was originally for cleanup of the temporary `.zip` we'd create for upload when calling `PrepareDependencies`.  this would now just remove the user's original dependencies each time